components/feature/addons/src/main/res/values/strings.xml

```diff
@@ -111,8 +111,8 @@
     <string name="mozac_feature_addons_updater_notification_title">%1$s has a new update</string>
     <!-- The content of the notification, this will be shown when an add-on needs new permissions, to be updated. %1$d is the amount of new permissions-->
     <string name="mozac_feature_addons_updater_notification_content" tools:ignore="PluralsCandidate">%1$d new permissions are required</string>
-    <!-- The content of the notification, this will be shown when an add-on needs a new permission-->
-    <string name="mozac_feature_addons_updater_notification_content_singular">A new permissions is required</string>
+    <!-- The content of the notification displayed/shown when an add-on needs a new permission-->
```

Comment:
I mean displayed/shown as an alternative, not to use both (just pick one) ;-)

Reply:
👍